src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.cs

```diff
@@ -151,6 +154,7 @@ protected virtual void GetServices()
             this.EditorAdaptersFactoryService = this.Package.ComponentModel.GetService<IVsEditorAdaptersFactoryService>();
             this.HostDiagnosticUpdateSource = this.Package.ComponentModel.GetService<HostDiagnosticUpdateSource>();
             this.AnalyzerFileWatcherService = this.Package.ComponentModel.GetService<AnalyzerFileWatcherService>();
+            this._featureServiceFactory = this.Package.ComponentModel.GetService<IFeatureServiceFactory>();
```

Comment:
Rather than fetching this upfront, we should fetch it along with the other things we're fetching in InstallStatementCompletion.